src/main/java/wooteco/subway/auth/application/AuthService.java

```diff
@@ -1,7 +1,46 @@
 package wooteco.subway.auth.application;
 
 import org.springframework.stereotype.Service;
+import wooteco.subway.auth.dto.TokenRequest;
+import wooteco.subway.auth.dto.TokenResponse;
+import wooteco.subway.auth.infrastructure.JwtTokenProvider;
+import wooteco.subway.exception.AuthorizationException;
+import wooteco.subway.exception.EmailNotFoundException;
+import wooteco.subway.member.dao.MemberDao;
+import wooteco.subway.member.domain.Member;
+
+import java.util.Optional;
 
 @Service
 public class AuthService {
+
+    private final JwtTokenProvider jwtTokenProvider;
+    private final MemberDao memberDao;
+
+    public AuthService(JwtTokenProvider jwtTokenProvider, MemberDao memberDao) {
+        this.jwtTokenProvider = jwtTokenProvider;
+        this.memberDao = memberDao;
+    }
+
+    private void validateToken(String token) {
+        if (jwtTokenProvider.validateToken(token)) {
+            return;
+        }
+        throw new AuthorizationException();
+    }
+
+    public Long LoginMemberId(String token) {
+        validateToken(token);
+        String id = jwtTokenProvider.getPayload(token);
+        return Long.parseLong(id);
+    }
+
+    public TokenResponse createToken(TokenRequest tokenRequest) {
+        Optional<Member> optionalMember = memberDao.findByEmail(tokenRequest.getEmail());
+        Member member = optionalMember.orElseThrow(EmailNotFoundException::new);
```

Comment:
따로 optionalMember를 만들지 않아도 될 것 같아요 :) 
```suggestion
        Member member = memberDao.findByEmail(tokenRequest.getEmail())
                .orElseThrow(EmailNotFoundException::new);
```

Reply:
이 부분도 다른 부분에서는 잘 만들어놓고 깜빡하고 고치지 않았네요 😮 🥊